TransformationPlayground/playground/transform_expert/parameters.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum
+
+"""
+This module contains enumerated types representing the separate axes that a transformation can be defined along.
+"""
+
+class SourceVersion(Enum):
+    ES_6_8 = "Elasticsearch 6.8"
+
+class TargetVersion(Enum):
+    OS_2_17 = "OpenSearch 2.17"
+
+class TransformType(Enum):
+    INDEX = "Index"
+
+class TransformLanguage(Enum):
+    PYTHON = "Python"
```

Comment:
Good callout.  This PR does Python because I expect it to be pretty easy to swap to the language we want (Jinja) and it seemed more important to get a first cut of this stuff out for folks to see than wait for the Jinja swap.  Basically, moving to Jinja will entail updating the section of our system prompt that currently defines our Python interface and some of the validation code for loading the transform into memory.  It \*should\* all be addressable with the current abstractions in place (tried to plan ahead a bit for it).